config/initializers/assets.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
+
+# Version of your assets, change this if you want to expire all your assets.
+Rails.application.config.assets.version = '1.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.